src/cdk-experimental/menu/menu-stack.ts

```diff
@@ -126,3 +126,19 @@ export class MenuStack {
     return this._elements[this._elements.length - 1];
   }
 }
+
+/** NoopMenuStack is a placeholder MenuStack used for inline menus. */
```

Comment:
Just to clarify -- does 'inline' mean it's triggered by some means other than a bar? (Such as a button?) I'd make this clearer in the documentation somewhere.

Reply:
Inline menus aren't triggered open at all - they're always open. Think of the list of mailboxes on gmail (inbox, starred, snoozed, ...).